### PR TITLE
Include image compression libraries

### PR DESCRIPTION
**Pull request by rick-nu**
### What

Include image compression libraries.
This is needed for front-end builders that compress images upon generation of the resources.

### CODE REVIEW

**Comment by rick-nu** [comment]
(Oh and I indented the apk libraries so that there is a more clear separation)

File: Dockerfile [hunk 1]
```diff
@@ -5,26 +5,39 @@ MAINTAINER docker@stefan-van-essen.nl
...
+        autoconf \
+        build-base \
+        zlib \
+        zlib-dev \
+        libpng \
+        libpng-dev \
+        libwebp \
+        libwebp-dev \
+        libjpeg-turbo \
+        libjpeg-turbo-dev \
```
Comment (eXistenZNL) [inline Dockerfile]: Please fix your crappy indenting of 4 spaces to the Glorious Master Indenting that is 2 spaces.

All hail 2 space indenting, ruler of the codebase!

Comment (rick-nu) [reply Dockerfile]: Oh fucking 2 spaces, I did this via the github code editor though. Anyway, will fix.

**Comment by rick-nu** [comment]
@eXistenZNL fixed!

**Pull request merged**